repos/system_upgrade/common/actors/checkconsumedassets/libraries/check_consumed_assets.py

```diff
@@ -0,0 +1,64 @@
+import re
+
+from leapp import reporting
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+from leapp.models import ConsumedDataAsset, LeappRepositoryInfo
+
+
+def inhibit_if_assets_with_incorrect_version():
+    leapp_info = next(api.consume(LeappRepositoryInfo), None)
+    if not leapp_info:
+        raise StopActorExecutionError('Did not receive any LeappRepositoryInfo messages.')
+
+    version_re = '([0-9]*)\\.([0-9]*)\\.([0-9]*)'
+    for consumed_asset in api.consume(ConsumedDataAsset):
+        if not consumed_asset.target_leapp_version:
+            # Most likely an old file that predates the introduction of versioning to data assets
+            summary = 'The asset {0} is missing target leapp version in its metadata'.format(consumed_asset.filename)
```

Comment:
```suggestion
            summary = 'The asset {0} is missing target Leapp version in its metadata header'.format(consumed_asset.filename)
```

Reply:
Resolved with the latest push as suggested.